orders/views.py

```diff
@@ -1,3 +1,140 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.utils     import timezone
+
+from users.utils     import LoginStatus
+from orders.models   import Order, OrderItem
+from products.models import ProductSize
+
+class OrdersPayment(View):
+    @LoginStatus
+    def post(self, request):
+        try:
+            id = request.user.id
+
+            datas = json.loads(request.body)
+
+            is_paid = Order.objects.filter(
+                id      = datas['orderId'], 
+                user_id = id, 
+                status  = 1
+            ).update(status_id = 2)
+            
+            if not is_paid:
+                return JsonResponse({'result' : 'INVALID PAYMENT'}, status=400)
+
+            return JsonResponse({'result' : 'SUCCESS'}, status=200)
+        except KeyError:
+            return JsonResponse({'result' : 'INVALID KEY'}, status=400)
+        except json.decoder.JSONDecodeError:
+            return JsonResponse({'result' : 'EMPTY BODY'}, status=400)
+
+class OrdersCart(View):
+    @LoginStatus
+    def get(self, request):
+        id = request.user.id
+
+        order = Order.objects.filter(user_id=id, status_id=1) or False
```

Comment:
사용자 별로 카트는 단일로 존재해야하기 get 을 사용하시는 것은 어떤가요?
모종의 이유로 2개 이상의 cart 상태의 order row 가 생길 수 있음을 가정한 것이라면, 
2개 이상일 때 get 을 사용하면 발생하는 에러 메시지를 확인해주시고, 그에 따른 예외처리를 해주시면 좋을 것 같습니다.
예외처리 :
1. row 가 없을 때
2. row 가 2개 이상일 때

```suggestion
        order = Order.objects.get(user_id=id, status_id=1)
```


추가적으로 filter 를 통해 받은 빈 쿼리셋은 그 자체가 falsy 한 값이기 때문에 or false 는 고려하지 않으셔도 괜찮을 것 같습니다.

Reply:
수정 완료하였습니다